iree/compiler/Conversion/LinalgToLLVMGPU/Common.cpp

```diff
@@ -0,0 +1,104 @@
+// Copyright 2021 Nod Labs
```

Comment:
I think these additional copyright notices might cause problems for us (at the least I will need to contact lawyers before proceeding :confused: ). IANAL, but my understanding from https://opensource.google/docs/releasing/contributions/#copyright is that the presence of them doesn't actually affect the contributors rights in any way, so if you're ok with just leaving this off, that would be my preference. Otherwise, I can contact our licensing team to discuss the options.

Reply:
Sure that's fine. I can remove the Nod Labs copyright headers from this patch.

Reply:
Removed my blocking review, but I'll let Thomas review for actual content